<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>255</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_26">
     <item>
      <widget class="QLabel" name="ficheiroLabel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nome do ficheiro:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ficheiroTextBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white
</string>
       </property>
       <property name="text">
        <string>Grelha</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_25">
     <item>
      <widget class="QLabel" name="cabecalhoLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor de fundo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cabecalhoButton">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="cabecalhoFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>134</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QLabel" name="periodosLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nº períodos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="periodosSpinBox">
       <property name="styleSheet">
        <string notr="true">background-color: white</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>218</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <widget class="QLabel" name="subseccoesLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nº subsecções:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="subseccoesSpinBox">
       <property name="styleSheet">
        <string notr="true">background-color: white</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QLabel" name="alunosLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nº alunos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="alunosSpinBox">
       <property name="styleSheet">
        <string notr="true">background:white
</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="gerarButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(217, 217, 217);
</string>
    </property>
    <property name="text">
     <string>Gerar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <widget class="QLabel" name="textoLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor do texto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="textoButton">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="textoFrame">
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>147</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QLabel" name="percentagemLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Percentagem:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="percentagemSpinBox">
       <property name="styleSheet">
        <string notr="true">background-color: white</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>255</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="QLabel" name="nomeLabel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nomeTextBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white
</string>
       </property>
       <property name="text">
        <string>Secção 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="seccoesLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Secções</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QLabel" name="textoAvaliacaoLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor do texto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="textoAvaliacaoButton">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="textoAvaliacaoFrame">
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_28">
     <item>
      <widget class="QLabel" name="cabecalhoAvaliacaoLabel">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor de fundo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cabecalhoAvaliacaoButton">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="cabecalhoAvaliacaoFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="seccoesLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Avaliação</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>160</y>
      <width>269</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="seccaoComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adicionarButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Adicionar secção</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removerButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Remover secção</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="carregarButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>79</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Carregar 
ficheiro</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
